src/puppetlabs/ezbake/core.clj

```diff
@@ -854,22 +854,25 @@ Additional uberjar dependencies:
 (defmethod action "build"
   [_ lein-project build-target]
   (action "stage" lein-project build-target)
+  (exec/exec "bundle" "install")
```

Comment:
We should explicitly set the dir we need the bundle install to install in so we can ensure we're running that command from the correct place.

```
(exec/exec "bundle" "install" :dir staging-dir)
```

Reply:
cool, I was going to ask about that. I saw that was used when we were bootstrapping, but wasn't sure exactly how that even worked..
will update.